openedxstats/apps/sites/views.py

```diff
@@ -27,8 +25,8 @@
 def SiteView_JSON(request):
     all_sites = Site.objects.all()
     active_sites = Site.objects.exclude(active_end_date__isnull=False)
-    active_sites_id_set = set(map(lambda site:site['id'], active_sites.values('id')))
-    country_list = list(map(lambda country:country['geo_zone'], SiteGeoZone.objects.values('geo_zone').order_by('geo_zone').distinct()))
+    active_sites_id_set = set([site['id'] for site in active_sites.values('id')])
```

Comment:
these changes were suggested by pycharm?

Reply:
Nope, it is done by 2to3

Reply:
@mraarif https://docs.python.org/2/library/2to3.html#2to3fixer-map